src/_assets/stylesheets/elements/_list.scss

```diff
@@ -0,0 +1,36 @@
+.claims-process-list {
+  @supports(counter-increment: item) {
```

Comment:
Making use of the [`@supports` CSS at-rule](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) here. Browsers that don't support `counter-increment` should display the page something like:

![screen shot 2017-05-30 at 11 02 34 am](https://cloud.githubusercontent.com/assets/27780860/26590108/c0d5aabe-4527-11e7-87df-e1ec4cabfdea.png)